src/plugins/stake-plugins/generic/policyAdapters/types.ts

```diff
@@ -15,5 +15,5 @@ export interface StakePolicyAdapter {
   fetchClaimQuote: (wallet: EdgeCurrencyWallet, requestAssetId: AssetId, nativeAmount: string) => Promise<ChangeQuote>
   fetchUnstakeExactQuote: (wallet: EdgeCurrencyWallet, requestAssetId: AssetId, nativeAmount: string) => Promise<ChangeQuote>
   fetchStakePosition: (wallet: EdgeCurrencyWallet) => Promise<StakePosition>
-  fetchYieldInfo: () => Promise<Pick<StakePolicy, 'apy' | 'yieldType'>>
+  fetchYieldInfo: () => Pick<StakePolicy, 'apy' | 'yieldType'>
```

Comment:
Let's keep this async. Our long-term plan is to move these plugins into the core, which forces all methods to be async because of the bridging. It doesn't cost much to return a no-op promise.